build/scripts-3.9/modelbuild.sh

```
@@ -0,0 +1,762 @@
+              -n GenericLabel \
+              -r ${_arg_output_dir}/${reg_type}/${i}/resample/$(basename ${_arg_inputs[${j}]}) \
+              -t ${_arg_output_dir}/${reg_type}/${i}/transforms/$(basename ${_arg_inputs[${j}]} | sed -r 's/(.nii$|.nii.gz$)//g')_1Warp.nii.gz \
+              -t ${_arg_output_dir}/${reg_type}/${i}/transforms/$(basename ${_arg_inputs[${j}]} | sed -r 's/(.nii$|.nii.gz$)//g')_0GenericAffine.mat \
+              -o ${_arg_output_dir}/${reg_type}/${i}/resample/masks/$(basename ${_arg_inputs[${j}]}) \
+              >> ${_arg_output_dir}/jobs/${_datetime}/${reg_type}_${i}_maskresample
+          fi
+        fi
+      done
+
+      # If masks were supplied, merge them and hard threshold at >25% confidence
+      if [[ " ${_arg_masks[@]} " =~ ".nii" ]]; then
```

Shellcheck (0.11.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)